backends/imgui_impl_dx11.cpp

```diff
@@ -484,7 +516,7 @@ bool    ImGui_ImplDX11_CreateDeviceObjects()
         g_pd3dDevice->CreateDepthStencilState(&desc, &g_pDepthStencilState);
     }
 
-    ImGui_ImplDX11_CreateFontsTexture();
+    ImGui_ImplDX11_UpdateFontsTexture();
```

Comment:
we doing same in NewFrame, shouldn't we delete this here?

Reply:
Work is being done on DX9 backend. Rest of them will be brought to sync after dust settle.